### ISSUE TITLE
[BUG] Regression: GestureDetector ignored on iOS

### ISSUE THREAD

**Issue by themadmrj** [jhomlala/betterplayer#570]
The code was working with v0.0.69, not working anymore with v0.0.70. 
I use the following code to pause my video onTap:
```
          GestureDetector(
              onTap: onTap,
              onLongPress: onLongPress,
              child: isVideo
                  ? BetterPlayer(controller: _betterPlayerController!))
```
To make it work on iOS, I simply use `BetterPlayerControlsConfiguration(playerTheme: BetterPlayerTheme.material)`, which made the trick. Sadly, as of v0.0.70, this does not work anymore on iOS and I need a replacement. Reverting to 0.0.69 fixed the problem. I already posted regarding this matter on #347. The fact that it happens again and that reverting to 0.0.69 fixes the problem, indicates it is not a Flutter problem, but indeed due to a change in the library.

EDIT: after further testing, `onLongPress` works fine, but `onTap` does not.

**Comment by LastxTemplar** [comment]
+1
this does seem to have to do with BetterPlayer

**Comment by jhomlala** [comment]
Fixed in 0.0.71. You need to use new special wrapper class: https://jhomlala.github.io/betterplayer/#/multiplegesturedetector
